custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -44,10 +44,21 @@ class WorkAnIssue(
         issuePage.waitForSummary()
     }
 
-    private fun edit(issuePage: AbstractIssuePage) {
+    private fun edit(issuePage: AbstractIssuePage, issueKey: String) {
         meter.measure(ActionType("Edit Issue Description") {Unit}) {
             issuePage.editDescription("updated")
         }
+        meter.measure(ActionType("Link Issue") {Unit}) {
```

Comment:
Do we want to link issues every time we edit? Do we want linking and editing to be coupled?
We can add an independent linking probability and method.